lib/widgets/action_sheet.dart

```diff
@@ -126,7 +134,7 @@ class BottomSheetHeaderPlainText extends StatelessWidget {
   }
 }
 
-/// A button in an action sheet.
+/// A button in a context menu.
 ///
 /// When built from server data, the action sheet ignores changes in that data;
```

Comment:
I think the term "action sheet" is going to remain in some places. The term "bottom sheet" will too, since it refers to the Material concept we're using.

So if we're going to introduce this third term, it'd be good to write down somewhere how the three different terms relate to each other. Perhaps on `_showContextMenu`? And/or here on ContextMenuItemButton, or have a reference here.